nextjs-app/shared/components/BlogCategoryFilter/BlogCategoryFilter.tsx

```
@@ -94,7 +94,7 @@ export function BlogCategoryFilter({
               <span
                 className={cn(
                   "ml-1.5",
-                  isActive ? "text-background/70" : "text-muted-foreground/70"
+                  isActive ? "text-background" : "text-muted-foreground"
```

ESLint

[error] Insert `,`

(prettier/prettier)